File: docs/source/how-tos/send-secure-msg/python/step3.py
async def init():
    me = input('Who are you? ').strip()
    wallet_config = '{"id": "%s-wallet"}' % me
    wallet_credentials = '{"key": "%s-wallet-key"}' % me

    # 1. Create Wallet and Get Wallet Handle
    try:
        await wallet.create_wallet(wallet_config, wallet_credentials)
    except:
        pass
    wallet_handle = await wallet.open_wallet(wallet_config, wallet_credentials)
    print('wallet = %s' % wallet_handle)

    (my_did, my_vk) = await did.create_and_store_my_did(wallet_handle, "{}")
    print('my_did and verkey = %s %s' % (my_did, my_vk))

    their = input("Other party's DID and verkey? ").strip().split(' ')
    return wallet_handle, my_did, my_vk, their[0], their[1]

File: docs/how-tos/send-secure-msg/rust/src/step3.rs
fn init() -> (i32, String, String) {
    let mut cmd = String::new();

    println!("Who are you? ");
    io::stdin().read_line(&mut cmd).unwrap();

    let config = json!({ "id" : format!("{}-wallet", cmd) }).to_string();
    wallet::create_wallet(&config, USEFUL_CREDENTIALS).wait().unwrap();
    let wallet_handle: i32 = wallet::open_wallet(&config, USEFUL_CREDENTIALS).wait().unwrap();

    let (did, verkey) = did::create_and_store_my_did(wallet_handle, "{}").wait().unwrap();
    println!("My DID and Verkey: {} {}", did, verkey);

    println!("Other party's DID and Verkey? ");
    let mut other = String::new();
    io::stdin().read_line(&mut other).unwrap();
    let other_verkey = other.trim().split(" ").collect::<Vec<&str>>()[1].trim().to_string();

    (wallet_handle, verkey, other_verkey)
}
